index.html

```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
       </ul>
     </nav>
   </header>
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
+    <figcaption>Dies ist ein Beispielbild</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <li><b>Phone: </b>+34 642 104 722</li>
+      <li><b>Mail: </b>danieleicher@gmx.net</li>
+    </ul>
+
+    <h3>Connect with me on</h3>
+
+    <!-- GitHub-logo with a link to my profil-->
+    <a href="https://github.com/codingoak" target="_blank"><img
+        src="https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg" width="32px" height="32px"
+        alt="github-logo"></a>
+    <!-- CodePen-logo with a link to my profil-->
+    <a href="https://codepen.io/codingoak" target="_blank"><img
+        src="https://res.cloudinary.com/css-tricks/images/w_32,h_32,c_fill/f_auto,q_auto/v1637703542/Button-Black-Large/Button-Black-Large-png?_i=AA"
+        width="32px" height="32px" alt="codepen-logo"></a>
+    <!-- LinkedIn-logo with a link to my profil-->
+    <a href="https://www.linkedin.com/in/daniel-eicher-963844203/" target="_blank"><img
+        src="https://upload.wikimedia.org/wikipedia/commons/c/ca/LinkedIn_logo_initials.png" width="32px" height="32px"
+        alt="linkedin-logo"></a>
```

Comment:
I googled it and it seems to be correct. However, I didn't do it yet :-)

Reply:
I'm not sure about the logos. There are copyrights and rules to use it...

Reply:
Well yeah, that's a good one. Maybe we can ask the team as well as the coaches tomorrow  :-)